### PR TITLE
Add travis.yml

### PR DESCRIPTION
**Pull request by 10sr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+language: go
+sudo: false
+go:
+  - 1.7.3
+  - 1.8.1
+  - tip
```
Comment (andreynering) [inline .travis.yml]: I think that if you use `1.7` and `1.8` it'll already use the latest minor version.

Also, not sure if we need to test `tip` here.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+matrix:
+  allow_failures:
+    - go: tip
+
+install:
+  - go get -u ./...
+  - go get -u gopkg.in/stretchr/testify.v1/assert
```
Comment (andreynering) [inline .travis.yml]: Add `-t` to `go get` to also download test dependencies. No need to separately get packages.

```sh
go get -u -t ./...
```

File: .travis.yml (hunk 1 above)
Comment (andreynering) [reply .travis.yml]: Ops, also Go latest version is `1.10` so I think:

```yml
go:
  - '1.8'
  - '1.9'
  - '1.10'
```

**Comment by 10sr** [comment]
Thank you! Fixed 🙌

**Comment by 10sr** [comment]
Now travis-ci has been enabled #3  and Test passed 👌

**Pull request merged**